extractor/src/extract/extractor.py

```diff
@@ -173,10 +173,9 @@ def run_sql_query(self, query, resource_id=None):
         return:
             the result of the sql query
         """
-        query = query.statement
-        logger.info(f"Executing query: {query}", extra={"resource_id": resource_id})
+        logger.info(f"Executing query: {query.statement}", extra={"resource_id": resource_id})
 
-        return self.session.execute(query)
+        return query.yield_per(10000)
```

Comment:
this constant should be defined someplace else (global scope, with a nice comment)